oci-library/oci-hol/java-management/set-up-of-management-agent/set-up-of-management-agent.md

```diff
@@ -284,12 +324,52 @@ Install Management Agent (If your host is Windows, skip to **For Windows** Secti
 
   ![image of modify agent settings page](/../images/fleet-modify-agent-settings-new.png)
 
+## Task 6: Verify application logging by Java Usage Tracker
+1. For the logging of applications to be visible, Java applications must be run again after the installation of the Management Agent. Run the following commands to run the HelloWorld application again:
+
+    ```
+    <copy>
+    javac HelloWorld.java
+    java HelloWorld
+    </copy>
+    ```
+
 11. If tagging and installation of management agents is successful, Java Runtimes will be indicated on the Fleet Main Page.
```

Comment:
probably good to leave a note here, telling users it will take a while for the java applications to appear in JMS. That is actually also tied to point 10 

_"Change the **Java Runtime Discovery** and **Java Runtime Usage** to the desired value."_

If user chose 5 mins, it takes 5 mins to update. if user chose 60mins, then it may take up to 60mins  to show up. I guess we can do them a favor by telling them to put 5 mins in Java Runtime usage in step 10?

Reply:
understood and will make the aforementioned changes!